src/main/webapp/script.js

```diff
@@ -129,4 +132,20 @@ function getZipCode() {
   xhttp.send();
 
   setTimeout(fetchList, 4000);
+}
+
+function initMap(mapLocation) {
+  let myLatLng = {lat: mapLocation.lat, lng: mapLocation.lng};
+  map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 13,
+    center: myLatLng,
+    mapTypeControl: false
+  });
+}
+
+function createMarker(listing) {
```

Comment:
There seems to be a discrepancy between the number of marker shown, and the number of results shown. Perhaps these should be the same?

Reply:
Now the map should only show the 15 markers that appear as listings.

Reply:
[deployed here!](https://step2020-smallcity.appspot.com)